workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -410,6 +420,45 @@ private boolean _hasEarlyPublishAccess(Jwt jwt) {
 		return false;
 	}
 
+	private boolean _hasIssuePermission(Jwt jwt, JSONObject issueJSONObject) {
+		JSONObject fieldsJSONObject = issueJSONObject.getJSONObject("fields");
+
+		String publishingStatus = fieldsJSONObject.optString(
+			"publishingStatus");
+
+		LocalDateTime publishingDate = _parsePublishingDate(
```

Comment:
can we move this `publishingDate` initialization inside the `publishingStatus.equals("Ready for Publishing")` if statement. that way if its not ready for publishing, none of this date logic has to execute

Reply:
Done!